src/index.html

```diff
@@ -5,8 +5,401 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__nav-list">
+          <li>
+            <a class="navigation__nav-link is-active">apple</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">samsung</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">smartphones</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link" data-qa="nav-hover">laptops & computers</a>
```

Comment:
Add href attribute to the anchor tag with a valid link or a placeholder link such as '#'.